src/API/HealthCheck.php

```diff
@@ -44,17 +43,17 @@ public function health_check() {
 
 		try {
 
+			$response = array();
+
 			if ( ! Pinterest_For_Woocommerce()::get_data( 'merchant_id' ) ) {
-				return array(
-					'status' => 'pending_initial_configuration',
-				);
+				$response['status'] = 'pending_initial_configuration';
+				return $response;
 			}
 
 			$merchant_connected_diff_platform = Pinterest_For_Woocommerce()::get_data( 'merchant_connected_diff_platform' );
 			if ( $merchant_connected_diff_platform ) {
-				return array(
-					'status' => 'merchant_connected_diff_platform',
-				);
+				$response['status'] = 'merchant_connected_diff_platform';
+				return $response;
```

Comment:
Similar suggestion as the above:

```suggestion
				return array( 'status' => 'merchant_connected_diff_platform' );
```